Sources/NIO/ChannelPipeline.swift

```diff
@@ -187,13 +187,14 @@ public final class ChannelPipeline: ChannelInvoker {
     /// May only be called from on the event loop.
     ///
     /// - parameters:
-    ///     - name: the name to use for the `ChannelHandler` when its added. If none is specified it will generate a name.
     ///     - handler: the `ChannelHandler` to add
+    ///     - name: the name to use for the `ChannelHandler` when it's added. If none is specified a name will be generated.
     ///     - position: The position in the `ChannelPipeline` to add `handler`. Defaults to `.last`.
     /// - returns: the result of adding this handler - either success or failure with an error code if this could not be completed.
-    private func _add(_ handler: ChannelHandler,
-                      name: String? = nil,
-                      position: ChannelPipeline.Position = .last) -> Result<Void, Error> {
+    @usableFromInline // should be fileprivate
```

Comment:
This `usableFromInline` is spurious, and so this function actually _can_ be fileprivate.